aws-sdk-core/spec/aws/sts/sts_regional_endpoints_spec.rb

```diff
@@ -27,15 +27,15 @@ module STS
           expect(client.config.sts_regional_endpoints).to eq('regional')
         end
 
-        it 'defaults to `legacy`' do
+        it 'defaults to `regional`' do
```

Comment:
These backticks should be removed too yeah?

Reply:
Yeah agreed.  These won't execute `regional` but I think better style would be: "defaults to 'regional'"